db/20230324145608.do.hdfstatus-trigger.sql

```diff
@@ -0,0 +1,12 @@
+CREATE TRIGGER [mtc_admin].[hdfStatusUpdatedAtTrigger]
```

Comment:
This sql is not in an actual migrations folder so is not actually being applied.  This is a good thing as there isn't a [hdfStatus] table.

Reply:
this migration has been completed and moved into the correct folder.